<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportAnimationDialog</class>
 <widget class="QDialog" name="ExportAnimationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Animation</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Animation type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBoxNumberOfSteps">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxAnimationType">
        <item>
         <property name="text">
          <string>time</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>viewpoint</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number of steps =</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Time span</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="horizontalSpacing">
       <number>9</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>to:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLabel" name="labelDiskSpaceRequired">
        <property name="text">
         <string>Space required = </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxStartIndex">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBoxEndIndex">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="labelNumberOfFiles">
        <property name="text">
         <string>Number of files = </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>From:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Viewpoint</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Rotate:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditRotate">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>deg</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Elevate:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditElevate">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>deg</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Output Files</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>File Prefix:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditFilePrefix">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Start Number:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditFileStartNumber">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Output Folder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEditOutputFolder"/>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>219</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonBrowse">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonPreview">
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonExport">
        <property name="text">
         <string>Export</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxStartIndex</tabstop>
  <tabstop>comboBoxEndIndex</tabstop>
  <tabstop>lineEditRotate</tabstop>
  <tabstop>lineEditElevate</tabstop>
  <tabstop>lineEditFilePrefix</tabstop>
  <tabstop>lineEditFileStartNumber</tabstop>
  <tabstop>lineEditOutputFolder</tabstop>
  <tabstop>pushButtonBrowse</tabstop>
  <tabstop>pushButtonExport</tabstop>
  <tabstop>pushButtonPreview</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>comboBoxAnimationType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
